### ISSUE TITLE
//taxcloud.net/imgs/cert.min.js doesn't work with jQuery >= v1.9

### ISSUE THREAD

**Issue by lat9** [taxcloud/ZenCart#8]
The subject jQuery module uses `$.browser`, which was deprecated in jQuery 1.9.  Is there any chance of getting a more recent jQuery version supported?

**Comment by taxcloud** [comment]
We will investigate, but this could be a false positive. Is the JS file actually loading?
Thanks.

**Comment by lat9** [comment]
Yes, it is.  I have installed the plugin on a fresh "load" of both Zen Cart 1.5.4 and 1.5.5f.  

Initially, I ran into an issue where a console log of `Reference error: babelHelpers is not defined` was being thrown by the module.  Looking at the plugin's `includes/templates/template_default/templates/tpl_taxcloud_exemptions_default.php`, I saw that it was loading jQuery 1.4.4 (which, by the way, is considered vulnerable by PCI scanners), so I removed that load.

After removing that jQuery 1.4.4 load (thus using the default zc1.5.4 jQuery 1.11.1), I'm met with a console log that indicates `Reference error: $.browser is not defined`.  That's due to that method's removal in jQuery 1.9 and later.

**Comment by wnzl17** [comment]
Am seeing same exemption certificate issue as lat9 on a zencart instance upgraded to 1.5.5f - Reference error: babelHelpers is not defined. I tried substituting loads of a range of jQuery versions above 1.4.4, but no success.
